src/features/steps/datsci.py

```diff
@@ -0,0 +1,76 @@
+import json
+import os
+
+from behave import given, then, when
+from helpers import (
+    aws_helper,
+    console_printer,
+    emr_step_generator,
+    file_helper,
+    invoke_lambda,
+)
+import operator
+
+CLUSTER_ARN = "ClusterArn"
+COMPLETED_STATUS = "COMPLETED"
+
+
+@given("I start the DATSCI cluster")
+def step_(context):
+    emr_launcher_config = {
+        "s3_overrides": None,
+        "overrides": None,
+        "extend": None,
+        "additional_step_args": None,
+    }
+
+    payload_json = json.dumps(emr_launcher_config)
+    console_printer.print_info(f"this is the payload: {payload_json}")
+    cluster_response = invoke_lambda.invoke_ap_launcher_lambda(payload_json)
+    print(cluster_response)
```

Comment:
could you remove this print or replace by console_printer.print_info is necessary?

Reply:
Removed. They were just extra print I added when debugging